xml/System.Globalization/CultureInfo.xml

```diff
@@ -153,15 +153,16 @@
 -   Standard cultures with user overrides. The user can use the **Region and Language** application in **Control Panel** to customize an existing culture's property values.  
   
 > [!NOTE]
->  You can use the <xref:System.Globalization.CultureAndRegionInfoBuilder> class to define, save, and register custom cultures that either supplement or replace existing cultures. The <xref:System.Globalization.CultureAndRegionInfoBuilder.Save%2A?displayProperty=nameWithType> method creates a Locale Data Markup Language (LDML) file that can be used to install a custom culture on target systems. For step-by step information on using the <xref:System.Globalization.CultureAndRegionInfoBuilder> class to create a new or replacement culture, see the <xref:System.Globalization.CultureAndRegionInfoBuilder> class topic.  
+>  We recommend you to use the [Microsoft Locale Builder](https://www.microsoft.com/en-us/download/details.aspx?id=41158) to register your custom cultures on the Windows operating system.
```

Comment:
>  We recommend you to use the [Microsoft Locale Builder](https://www.microsoft.com/en-us/download/details.aspx?id=41158) to register your custom cultures on the Windows operating system.

 We recommend you to use the [Microsoft Locale Builder](https://www.microsoft.com/en-us/download/details.aspx?id=41158) to create and register your custom cultures on the Windows operating system.